src/pages/workspaces/workspace/Data.js

```diff
@@ -314,30 +351,59 @@ class extends Component {
   renderGlobalVariables() {
     const { namespace } = this.props
     const { workspaceAttributes } = this.state
+    const filteredAttributes = _.flow(
+      _.toPairs,
+      _.remove(([key]) => key === 'description' || key.includes(':') || key.startsWith('referenceData-')),
+      _.sortBy(_.first)
+    )(workspaceAttributes)
 
     return Utils.cond(
-      [!workspaceAttributes, () => undefined],
-      [_.isEmpty(workspaceAttributes), () => 'No Global Variables defined'],
+      [!filteredAttributes, () => undefined],
+      [_.isEmpty(filteredAttributes), () => 'No Global Variables defined'],
       () => h(AutoSizer, { disableHeight: true }, [
         ({ width }) => h(FlexTable, {
-          width, height: 500, rowCount: workspaceAttributes.length,
+          width, height: 500, rowCount: filteredAttributes.length,
           columns: [
             {
               size: { basis: 400, grow: 0 },
               headerRenderer: () => h(HeaderCell, ['Name']),
-              cellRenderer: ({ rowIndex }) => renderDataCell(workspaceAttributes[rowIndex][0], namespace)
+              cellRenderer: ({ rowIndex }) => renderDataCell(filteredAttributes[rowIndex][0], namespace)
             },
             {
               size: { grow: 1 },
               headerRenderer: () => h(HeaderCell, ['Value']),
-              cellRenderer: ({ rowIndex }) => renderDataCell(workspaceAttributes[rowIndex][1], namespace)
+              cellRenderer: ({ rowIndex }) => renderDataCell(filteredAttributes[rowIndex][1], namespace)
             }
           ]
         })
       ])
     )
   }
 
+  renderReferenceData() {
+    const { namespace } = this.props
+    const { selectedDataType } = this.state
+    const selectedData = _.sortBy('0', this.getReferenceData()[selectedDataType])
```

Comment:
I have no doubt that this works, but perhaps sorting by `_.first` would be clearer?